src/main/kotlin/com/vk/admstorm/ssh/SshConnectionService.kt

```diff
@@ -123,6 +125,14 @@ class SshConnectionService(private var myProject: Project) : Disposable {
         // Disconnect the current connection, if it exists.
         disconnect()
 
+        val scheduler = ConcurrencyUtil.newSingleScheduledThreadExecutor("YubiKeyTouchingNotification")
+        scheduler.schedule({
+            ApplicationManager.getApplication().invokeLater {
```

Comment:
You can just use the `invokeLater` from `com.intellij.openapi.application.invokeLater`

Reply:
This is `Obsolete`